### ISSUE TITLE
Bogus message (from another package)?

### ISSUE THREAD

**Issue by scop** [rebase-helper/rebase-helper#211]
https://bugzilla.redhat.com/show_bug.cgi?id=1347160#c20

There is no file named `libtasn1-3.4-rpath.patch` in the python-flake8 package, nor has ever been. There is one in the libtasn1 package though. The system producing these messages seems quite badly confused.

**Comment by phracek** [comment]
@nforro Can you please have a look at it? I guess, it is already solved, right? version rebase-helper-0.7.3-2.

**Comment by nforro** [comment]
@phracek: Yes, this seems to be the same issue as #174 and it should be fixed in rebase-helper-0.7.3-2. What rebase-helper version is running on upstream monitoring service? Too bad we can't see version number in the attached log.
